README.md

```diff
@@ -1,6 +1,11 @@
 
 # fredapi: Python API for FRED (Federal Reserve Economic Data)
 
+[![Build and test GitHub](https://github.com/mortada/fredapi/actions/workflows/main.yml/badge.svg)](https://github.com/mortada/fredapi/actions)
+[![version](https://img.shields.io/badge/version-0.5.1-success.svg)](#)
```

Comment:
just realized this is actually hardcoded to `0.5.1`, any idea how to update this, or maybe this should be removed because the latest release is already shown by the line below? thanks!

Reply:
cc @LucaMingarelli

Reply:
@mortada : ideally they would always show the same version, but they relate to different things: the former is the version on github, the second is the version actually published on pypi. If they differ they serve as a reminder that the latest version available has not been published yet. 

One option is of course to update the badge manually, however this https://github.com/mortada/fredapi/compare/master...LucaMingarelli:fredapi:patch-1 can also be used within the workflow to have it done automatically. 

It needs to be called from the main.yml like so:
https://github.com/mortada/fredapi/compare/master...LucaMingarelli:fredapi:patch-2

Reply:
Uff, there's a typo on this https://github.com/mortada/fredapi/compare/master...LucaMingarelli:fredapi:patch-1: line 25 should be `f.write(README_NEW)` ...